src/task/task.js

```diff
@@ -0,0 +1,51 @@
+const randomId = require("../utils/randomId");
+
+const getTasks = async (apiClient, { projectId }) => {
+  try {
+    const {
+      data: {
+        changes: {
+          tasks: { updated },
+        },
+      },
+    } = await apiClient.get("sync", {
+      collection_name: "tasks",
+    });
+
+    if (projectId) {
+      return updated.filter((task) => task.project_id === projectId);
+    }
+
+    return updated;
+  } catch (error) {
+    console.error(error);
```

Comment:
why?

Reply:
Well, I'm still wondering how the errors should be handled, that's why I am just logging the error. You've mentioned that

> this API is designed for reliable synchronization, not REST API-l'ike usage, so its behavior is unusual. It almost never fails, and nearly all errors are silent -- e.g. missing/incorrect fields will be sanitized to correct values on "best effort" basis instead of informing the API user of an error.

However there might be an error thrown if the particular access token is unauthorized (401). The user should be aware of the wrong token, I suppose. Maybe there are some other cases I am not aware of. 

Maybe all the `catch`es should return a custom error, or just the the status code?